staging/src/k8s.io/client-go/tools/leaderelection/leaderelection_test.go

```diff
@@ -352,3 +341,580 @@ func TestLeaseSpecToLeaderElectionRecordRoundTrip(t *testing.T) {
 		t.Errorf("diff: %v", diff.ObjectReflectDiff(oldRecord, newRecord))
 	}
 }
+
+func multiLockType(objectType string) (primaryType, secondaryType string) {
+	switch objectType {
+	case rl.EndpointsLeasesResourceLock:
+		return rl.EndpointsResourceLock, rl.LeasesResourceLock
+	case rl.ConfigMapsLeasesResourceLock:
+		return rl.ConfigMapsResourceLock, rl.LeasesResourceLock
+	default:
+		panic("unexpected objType:" + objectType)
```

Comment:
instead of panic-ing, can you pass testing.T as an argument and call t.Fatalf() ?

Reply:
Same below

Reply:
Just following the code above `func createLockObject`.

Reply:
I will fix them all.